src/Indice.Features.Messages.App/src/environments/environment.ts

```diff
@@ -6,7 +6,7 @@ export const environment = {
   api_url: 'https://localhost:2001/api',  //'https://messaging.indice.gr/api',
   auth_settings: {
     accessTokenExpiringNotificationTime: 60,
-    authority: 'https://my.indice.gr',
+    authority: 'https://localhost:2000',
```

Comment:
This is irrelevant to the feature. Do not change arbitrary settings according to debug preferences.